src/amber/cli/generators/auth_base.cr

```diff
@@ -0,0 +1,116 @@
+require "./generator"
+require "./field"
+
+module Amber::CLI
+  class AuthBase < Generator
```

Comment:
Maybe AuthGenerator?

Reply:
I prefer base myself.  There is an Auth that provides the command and picks granite or crecto.  When we move that out to its own shard (if that is possible) we will eliminate the need for this base class.

Reply:
I asked cause I saw Auth and AuthBase